screens/EventCreation/EventCreationScreen.tsx

```diff
@@ -12,10 +12,24 @@ import { useNavigation } from "@react-navigation/native"
 import { Ionicons } from "@expo/vector-icons"
 import { globalStyles } from "../../assets/global/globalStyles"
 import { peach, white } from "../../assets/colors/colors"
+import { createEvent } from "../../firebase/ManageEvents"
 
 const EventCreationScreen = () => {
   const navigation = useNavigation()
   const [isEvent, setIsEvent] = useState(false)
+  const [title, setTitle] = useState("")
+  const [description] = useState("lore ipsum")
+  const [location] = useState("Besac City")
+  // const [startDate, setStartDate] = useState<Date>()
+  // const [endDate, setEndDate] = useState("")
+  // const [imageUrl, setImageUrl] = useState("")
+//  const [point, setPoint] = useState({ x: 47.238458, y: 5.984155 })
+
+
+  const newEvent = async () => {
+    console.log("Creating event")
+    await createEvent("uid2", title, description, new Date(2025, 0, 1), { x: 47.238458, y: 5.984155 }, location, "imageUrl")
```

Comment:
You could add a navigation back here I think, when the event is successfully created.

Reply:
Good point. I'll let @AlbertoCentonze , who is designing the screen do it.